src/System.Private.Xml/src/System/Xml/Xsl/IlGen/XmlILModule.cs

```diff
@@ -138,10 +135,7 @@ public XmlILModule(bool useLRE, bool emitSymbols)
                 }
 
                 // Create ModuleBuilder
-                if (_persistAsm)
-                    modBldr = asmBldr.DefineDynamicModule("System.Xml.Xsl.CompiledQuery");
-                else
-                    modBldr = asmBldr.DefineDynamicModule("System.Xml.Xsl.CompiledQuery"); //, this.modFile + ".dll", emitSymbols); //BinCompat TODO
```

Comment:
You are absolutely right. I noticed this was switched by myself mistakenly when I was porting this. You can see the Desktop version is the other way: http://index/?rightProject=System.Data.SqlXml&file=System%5cXml%5cXsl%5cIlGen%5cXmlILModule.cs&line=150